docs/Containers/WireGuard.md

```diff
@@ -103,9 +103,60 @@ With most containers, you can continue to tweak environment variables and settin
 
 #### <a name="configurePeerDNS"> Optional configuration - DNS resolution for peers </a>
 
-* `PEERDNS=`. The default value of `auto` instructs the WireGuard container to use the same DNS as other containers when resolving requests from connected peers. In practice, that means the container directs queries to 127.0.0.11, which Docker intercepts and forwards to whichever resolvers are specified in the Raspberry Pi's `/etc/resolv.conf`.
+You have several options for how your remote peers resolve DNS requests:
+
+* `PEERDNS=auto`
+
+	The default value of `auto` instructs the WireGuard *service* running within the WireGuard *container* to use the same DNS as the WireGuard *container* when resolving requests from connected peers. In practice, that means the *service* directs queries to 127.0.0.11, which Docker intercepts and forwards to whichever resolvers are specified in the Raspberry Pi's `/etc/resolv.conf`.
```

Comment:
Not exactly. PEERDNS=auto will use the **coredns** daemon running in the wireguard container, not the "docker container itself". Coredns is per default configured to use /etc/resolv.conf (from inside the container) which tells it to use 127.0.0.11 as the nameserver.